Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM docker.io/centos:centos8
 
-ARG PKGS_LIST=main-packages-list.txt
+ENV PKGS_LIST=main-packages-list.txt
+ARG EXTRA_PKGS_LIST
 
-COPY ${PKGS_LIST} /tmp/main-packages-list.txt
+COPY ${PKGS_LIST} ${EXTRA_PKGS_LIST} /tmp/
```

Comment:
If EXTRA_PKGS_LIST is left blank this results in the entire contents of $PWD being copied to /tmp, I know its a podman bug but it would be nice to avoid it if possible.

Reply:
the only way that I could think about at this moment would be having a placeholder, like an empty default file
unfortunately we can't use COPY conditionally :/

Reply:
Let's use a non-existing file to indicate an error?

Reply:
> Let's use a non-existing file to indicate an error?

I'm not sure I understand, would you mind expanding your thought?

Reply:
Just defaulting EXTRA_PKGS_LIST to `/Please/set/EXTRA_PKGS_LIST`? Or indeed create an empty file to default to?

Reply:
Or defaulting EXTRA_PKGS_LIST to PKGS_LIST, yes it can result in the COPY copying the same file twice but works
```
-COPY ${PKGS_LIST} ${EXTRA_PKGS_LIST} /tmp/
+COPY ${PKGS_LIST} ${EXTRA_PKGS_LIST:-$PKGS_LIST} /tmp/

```

Reply:
@derekhiggins I did a test and using that line seems to work
going to change that